<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPreference</class>
 <widget class="QDialog" name="DialogPreference">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>341</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>端口</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="cbDevices"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>波特率</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="cbBaudRate">
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>38400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>57600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>数据位</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="cbDataBits">
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>校验位</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="cbParity">
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Even</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Odd</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mark</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Space</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>停止位</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="cbStopBits">
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1.5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>流控</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="cbFlowControl">
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RTS/CTS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>XON/XOFF</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pbSave">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>520</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>保存配置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>520</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
